documentation/faq.mdx

```diff
@@ -50,16 +50,16 @@ This is caused by a false-positive detection, which we can do nothing about. If
 
 ### How finished is the project?
 
-The first game is at a stage where it is mostly feature complete and bug free on the default settings. _Jak II_ is in very early stages, come back later!
+The games _Jak and Daxter: The Precursor Legacy_ and _Jak II_ are at a stage where they are feature complete and bug free on the default settings. _Jak II_ is as of writing still in beta due to a few background music issues, which shouldn't get in the way of completing the game. _Jak 3_ is in very early stages, come back later!
```

Comment:
The issues don't pertain to background music, it's better to just say "mainly a few audio and graphics issues."